src/components/AppNavHeader.tsx

```diff
@@ -30,9 +30,10 @@ export function AppNavHeader({
     route,
     styles.titleText,
   );
+  const statusBarHeight = config.AppNavHeader?.statusBarHeight;
```

Comment:
Yes, it might.  Currently we are relying on [Paper's AppBar](https://github.com/callstack/react-native-paper/blob/main/src/components/Appbar/AppbarHeader.tsx) which uses safe-area-view-context to add margin there (or not).  Looking [at the code](https://github.com/th3rdwave/react-native-safe-area-context/blob/main/src/SafeAreaContext.tsx), I it using `Dimensions` from react-native, vs the library you linked which [uses presets for various ios hardware](https://www.npmjs.com/package/react-native-status-bar-height?activeTab=code) and I'm not sure which is better.

Specifically, with "statusBarHeight" above, we currently only need to set this to `0` to turn it off, or leave it `undefined` to have the default margin set by react-native-safe-area-context.

Reply:
Gotcha, lets stick with what you have then. I think the package I suggested is too old anyways and the safe area context should be all we need.